Sources/PandaSDK/Extensions/Extensions+Collection.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  File.swift
+//  
+//
+//  Created by Nick Nick  on 11/10/22.
+//
+
+import Foundation
+
+extension Collection where Iterator.Element == [String: Any] {
+    func toJSONString(options: JSONSerialization.WritingOptions = .fragmentsAllowed) -> String {
+    if let array = self as? [[String: Any]],
+       let data = try? JSONSerialization.data(withJSONObject: array, options: options),
```

Comment:
Why do u need to cast self as array if `JSONSerialization.data(withJSONObject:` use `Any`.

Reply:
Because I apply it in case  object is  [[String: Any]] only.